tests/typ/bugs/equation-numbering-reference.typ

```diff
@@ -0,0 +1,24 @@
+// In this bug, the hint and error messages for an equation
+// being reference mentioned that it was "header" and was
+// lacking the proper path.
+
+---
+#set page(height: 70pt)
+
+$
+    Delta = b^2 - 4 a c
+$ <quadratic>
+
+// Error: 14-24 cannot reference equation without numbering
+// Hint: 14-24 you can enable math.equation numbering with `#set math.equation(numbering: "1.")`
+Looks at the @quadratic formula.
+
+---
+#set page(height: 70pt)
```

Comment:
Why the second test case? Isn't there a test for equation references somewhere already?

Reply:
Fair enough, I just assumed it was good practice to include the "well behaved" test when testing bugs